docs/manual/en/introduction/Installation.html

```diff
@@ -88,42 +88,53 @@ <h2>Install from CDN or static hosting</h2>
 		</code>
 
 		<p>
-			Since Import maps are not yet supported by all browsers, it is necessary to add the polyfill *es-module-shims.js*.
+			Since import maps are not yet supported by all browsers, it is necessary to add the polyfill *es-module-shims.js*.
 		</p>
 
-		<h2>Examples</h2>
+		<h2>Addons</h2>
 
 		<p>
-			The core of three.js is focused on the most important components of a 3D engine. Many other useful components — such as controls, loaders, and post-processing effects — are part of the [link:https://github.com/mrdoob/three.js/tree/dev/examples/jsm examples/jsm] directory. They are referred to as "examples," because while you can use them off the shelf, they're also meant to be remixed and customized. These components are always kept in sync with the core library, whereas similar third-party packages on npm are maintained by different people and may not be up to date.
+			The core of three.js is focused on the most important components of a 3D engine. Many other useful components — such as controls, loaders, and post-processing effects — are part of the [link:https://github.com/mrdoob/three.js/tree/dev/examples/jsm examples/jsm] directory. They are referred to as "addons" (previously called "examples"), because while you can use them off the shelf, they're also meant to be remixed and customized. These components are always kept in sync with the core library, whereas similar third-party packages on npm are maintained by different people and may not be up to date.
 		</p>
 
 		<p>
-			Examples do not need to be <em>installed</em> separately, but do need to be <em>imported</em> separately. If three.js was installed with npm, you can load the [page:OrbitControls] component with:
+			Addons do not need to be <em>installed</em> separately, but do need to be <em>imported</em> separately. If three.js was installed with npm, you can load the [page:OrbitControls] component with:
 		</p>
 
-
 		<code>
 		import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
 
 		const controls = new OrbitControls( camera, renderer.domElement );
 		</code>
 
 		<p>
-			If three.js was installed from a CDN, use the same CDN to install other components:
+			If three.js was installed from a CDN, use the same code, but with `three/addons/` in the import map.
 		</p>
 
 		<code>
+		&lt;script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js">&lt;/script>
+
+		&lt;script type="importmap">
+		  {
+		    "imports": {
+		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
+                      "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```

Comment:
```suggestion
		    "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```

Hey @LeviPesin looks like this is wrongly formatted?

<img width="933" alt="image" src="https://user-images.githubusercontent.com/7217420/188615010-6605e033-ea6e-4ce1-8ab1-fac803170cf1.png">

Reply:
still broken, I'll let you fix properly hehe 

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/7217420/188623655-afd6afc8-dc9c-4cf9-88e1-bed077b91c16.png">